x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -275,6 +281,28 @@ export const SearchBar: FC<SearchBarProps> = (opts) => {
 
   useEvent('keydown', onKeyDown);
 
+  const loadNoResult = useCallback(() => {
```

Comment:
Do we still need these changes?

Reply:
Yeah good call, loading state is working and this doesn't seem necessary. [89ee4b7](https://github.com/elastic/kibana/pull/183866/commits/89ee4b7b7a5f8b4af2bdc962f36bd6c946c00190) for changes, thanks!